scripts/config.txt

```diff
@@ -0,0 +1,7 @@
+#Clear Linux version to use as base rootfs
```

Comment:
I think this should be part of runtime versions.txt, right?

Reply:
Yes, I feel that is a better idea. instead of having to maintain multiples versions files I can point or could to know what is the version to use,  reworking that.